native/core/src/execution/datafusion/planner.rs

```diff
@@ -1599,6 +1599,18 @@ impl PhysicalPlanner {
         }
     }
 
+    /// Find DataFusion's built-in window function by name.
+    fn find_df_window_function(name: &str) -> Option<WindowFunctionDefinition> {
+        if let Some(f) = find_df_window_func(name) {
+            Some(f)
+        } else {
+            match name {
```

Comment:
Do we need to do `name.to_lowercase()`?

Reply:
I think we are OK. `name` is from this [method](https://github.com/apache/datafusion-comet/blob/04ceba8661b4e402a41586c51c8f340389fb8116/native/core/src/execution/datafusion/planner.rs#L1588). I have lower case `count`, `min`, `max` for all of them.